content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -32,7 +32,7 @@ The cluster that `kubeadm init` and `kubeadm join` set up should be:
    - `kubeadm init`
    - `export KUBECONFIG=/etc/kubernetes/admin.conf`
    - `kubectl apply -f <network-of-choice.yaml>`
-   - `kubeadm join --token <token> <master-ip>:<master-port>`
+   - `kubeadm join --token <token> <api-server-hostname>:<port>`
```

Comment:
better to call this `<endpoint>:<port>` because join works with "controlplane endpoint" or "apiserver endpoint"

Reply:
👍